js/app.js

```diff
@@ -0,0 +1,224 @@
+  }
+
+  return htmlTemplate;
+}
+
+let worksCardsButtons;
+let quitPopUp;
+const worksPopUpSection = document.getElementById('works-pop-up');
+
+document.addEventListener('DOMContentLoaded', () => {
+  const works = document.getElementById('trabajos');
+
+  projects.forEach((n) => {
+    works.innerHTML += generateProjectCard(n);
+  });
+
+  worksCardsButtons = document.querySelectorAll('.tarjeta .grupo-boton button');
+
+  worksCardsButtons.forEach((u) => {
+    u.addEventListener('click', () => {
+      const id = u.getAttribute('id');
+      for (let i = 0; i < projects.length; i += 1) {
+        if (id === projects[i].buttonId) {
+          // eslint-disable-next-line no-use-before-define
+          worksPopUpSection.innerHTML += worksPopUp(projects[i]);
+        }
+      }
+      quitPopUp = document.querySelector('.tarjetas-popup .exit-boton');
+      quitPopUp.addEventListener('click', () => {
+        worksPopUpSection.innerHTML = '';
+      });
+    });
+  });
+});
+
+// POPUP
+
+function worksPopUp(project) {
+  let htmlTemplate = '';
+  htmlTemplate = `
+  <div class="fondo-popup">
+  <div class="flex-column tarjetas-popup ">
+      <div class="flex-row arriba ">
+          <h3 class="articulo-trabajo">
+              ${project.name}
+          </h3>
+          <button class="exit-boton">
+          </button>
+      </div>
+      <ul class="empresa articulo-trabajo flex-row">
+          <li>
+              <h4>${project.company.name}</h4>
+          </li>
+          <li><img src="media/Counter.png"></li>
+          <li>
+              <p>${project.company.position}</p>
+          </li>
+          <li><img src="media/Counter.png"></li>
+          <li>
+              <p>${project.company.date}</p>
+          </li>
+      </ul>
+      <div class="popup-imagen-grupo">
+      <img class="foto-popup articulo-trabajo" src="${project.image}" alt="">
+      </div>
+      <div class="escritorio-capa-popup">
+          <p class="tarjeta-explicacion articulo-trabajo">
+              ${project.description}
+          </p>
+          <div class="derecha lista">
+              <ul class="lista indices articulo-trabajo flex-row">
+                  ${generateTechnologyList(project.technology)}
+              </ul>
+              <hr>
+              <div class="flex-row button-container">
+                 <div class="grupo-boton">
+                   <a href="https://maximoortelli.github.io/Portfolio-setup-and-mobile--first/" target="_blank">
+                       <button class="flex-row">See live <img class="button-icon" src="media/See live icon.png"></button>
+                   </a>
+                 </div>
+                 <div class="grupo-boton">
+                   <a href="https://github.com/maximoortelli/Portfolio-setup-and-mobile--first/tree/Portfolio-Pop-Up-Window" target="_blank">
+                     <button class="flex-row">See source <img class="button-icon" src="media/See source icon.png"></button>
+                   </a>
```

Comment:
- [ ] 👏🌟 Good progress on updating the button styles for the desktop version! 🚀💻 However, there are still some adjustments needed to match the design requirements. 👀🎨 The current button styles don't quite align with the design, and I recommend paying closer attention to the design and updating your code to match the design.

![image](https://github.com/maximoortelli/Portfolio-setup-and-mobile--first/assets/55840999/c53fcf38-cd88-4a8a-a9d7-0841e9a13c71)

By fine-tuning these button styles and adhering closely to the design, your project will look even more polished and professional! 🎉💪 Keep up the good work, and happy coding! 💻🚀

Reply:
Thank you very much for advising me, I hope that now with all these changes you can approve me. I have completed all revisions satisfactorily 👍